### PR TITLE
clever.service: pipe all errors to /tmp/clever.err

### PR DESCRIPTION
**Pull request by okalachev**
* pipe all service errors to `/tmp/clever.err`
* show all service error in selfcheck.py, remove `python-systemd` usage

### CODE REVIEW

File: builder/image-software.sh [hunk 1]
```diff
@@ -107,7 +107,6 @@ espeak espeak-data python-espeak \
 ntpdate \
 python-dev \
 python3-dev \
-python-systemd \
```
Comment (sfalexrog) [inline builder/image-software.sh]: Wow, that's fantastic!

File: clever/src/selfcheck.py [hunk 2]
```diff
@@ -630,31 +630,10 @@ def check_clever_service():
         return
 
     try:
-        from systemd import journal
```
Comment (sfalexrog) [inline clever/src/selfcheck.py]: Really love dropping this dependency. Way to go!

File: clever/src/selfcheck.py [hunk 3]
```diff
@@ -630,31 +630,10 @@ def check_clever_service():
...
-        elif ('[ERROR]' in msg) or ('[FATAL]' in msg):
-            msg = r.search(msg).groups()[2]
-            if msg in node_errors:
-                continue
-            node_errors.append(msg)
-        elif ('ERROR: ' in msg) or ('while processing' in msg) or ('Invalid roslaunch XML syntax' in msg):
-            node_errors.append(msg)
-    for error in reversed(node_errors):
-        failure(error)
+        for line in open('/tmp/clever.err', 'r'):
```
Comment (sfalexrog) [inline clever/src/selfcheck.py]: Well, we're losing deduplication, among other things. Have you tried this with actual node/nodelet errors?

Comment (okalachev) [reply clever/src/selfcheck.py]: Fixed.

**Comment by okalachev** [comment]
Screenshot 2019-09-05 at 20 35 42

File: clever/src/selfcheck.py [hunk 4]
```diff
@@ -631,9 +632,26 @@ def check_clever_service():
...
+                else:
+                    error_count.update({msg: 1})
+            else:
+                error_count.update({line.strip(): 1})
+
+        for error in error_count:
+            if error_count[error] == 1:
+                failure(error)
+            else:
+                failure('%s (%s)', error, error_count[error])
```
Comment (sfalexrog) [inline clever/src/selfcheck.py]: Shouldn't the format specifier be `'%s (%d)'`? I'm pretty sure this works as well, it's just that the error count is acutally an integer.

File: clever/src/selfcheck.py [hunk 5]
```diff
@@ -631,9 +632,26 @@ def check_clever_service():
     elif 'failed' in output:
         failure('service failed to run, check your launch-files')
 
+    r = re.compile(r'^(.*)\[(FATAL|ERROR)\] \[\d+.\d+\]: (.*?)(\x1b(.*))?$')
```
Comment (sfalexrog) [inline clever/src/selfcheck.py]: So much for simplier code, I guess.

**Pull request merged**